src/core/structure.js

```diff
@@ -245,3 +242,12 @@ function createTableOfContents(ol) {
   const link = hyperHTML`<p role='navigation' id='back-to-top'><a href='#title'><abbr title='Back to Top'>&uarr;</abbr></a></p>`;
   document.body.append(link);
 }
+
+function appendixNumber(index) {
```

Comment:
Can you confirm that we want 26 to map to 'BA' instead of 'AA'?
I added some tests based on how spreadsheets handle index to column name mapping, which I think is the desirable.

``` js
const tests = [
  [0, "A"],
  [1, "B"],
  [2, "C"],
  [25, "Z"],
  [26, "AA"],
  [27, "AB"],
  [28, "AC"],
  [54, "BC"]
];

for (const [index, expected] of tests) {
  const result = appendixNumber(index);
  console.assert(result === expected, "", { index, result, expected });
}
```

Reply:
I wrote a version which passes above tests. WDYT?
``` js
function appendixNumber(index) {
  index += 1; // index is 0 based
  let s = "";
  while (index > 0) {
    index -= 1;
    s = String.fromCharCode(65 + (index % 26)) + s;
    index = Math.floor(index / 26);
  }
  return s;
}
```